drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -0,0 +1,233 @@
+local security = require "st.security"
+local ZigbeeDriver = require "st.zigbee"
+local data_types = require "st.zigbee.data_types"
+local cluster_base = require "st.zigbee.cluster_base"
+local capabilities = require "st.capabilities"
+local base64 = require "base64"
+local credential_utils = require "credential_utils"
+
+local remoteControlStatus = capabilities.remoteControlStatus
+local lockCredentialInfo = capabilities["stse.lockCredentialInfo"]
+local Battery = capabilities.battery
+local Lock = capabilities.lock
+local TamperAlert = capabilities.tamperAlert
+
+local PRI_CLU = 0xFCC0
+local PRI_ATTR = 0xFFF3
+local MFG_CODE = 0x115F
+
+local serial_num = 0
+local seq_num = 0
+
+local SHARED_KEY = "__shared_key"
+local CLOUD_PUBLIC_KEY = "__cloud_public_key"
+
+local function my_secret_data_handler(driver, device, secret_info)
+  -- At time of writing this returns nothind beyond "secret_type = aqara"
+  local shared_key = secret_info.shared_key
+  local cloud_public_key = secret_info.cloud_public_key
+
+  device:set_field(SHARED_KEY, shared_key, { persist = true })
+  device:set_field(CLOUD_PUBLIC_KEY, cloud_public_key, { persist = true })
+  credential_utils.save_data(driver)
+
+  if cloud_public_key ~= nil then
+    local raw_data = base64.decode(cloud_public_key)
+    -- send cloud_pub_key
+    device:send(cluster_base.write_manufacturer_specific_attribute(device,
+      PRI_CLU, PRI_ATTR, MFG_CODE, data_types.OctetString, "\x3E" .. raw_data))
+  end
+end
+
+local function device_added(self, device)
+  if credential_utils.is_exist_host(device) then
+    device:emit_event(remoteControlStatus.remoteControlEnabled('true', { visibility = { displayed = false } }))
```

Comment:
why isn't this value's datatype a `boolean`?

Reply:
it's exisitng standard capability. this code just use that.

Reply:
https://github.ecodesamsung.com/iot/SmartThingsCapabilities/blob/8f6538bf03eacc140c38954299184bde9599886c/capabilities/remote-control-status.src/remote-control-status.yml#L10

It seems that definition of the `remoteControlEnabled` is string.